src/IDelegateRegistry.sol

```diff
@@ -13,37 +13,59 @@ interface IDelegateRegistry {
         NONE,
         ALL,
         CONTRACT,
-        ERC721,
         ERC20,
+        ERC721,
         ERC1155
     }
 
-    /// @notice Info about a single delegation, used for onchain enumeration
-    struct DelegationInfo {
+    /// @notice Standardizes storage positions of delegation data
+    enum StoragePositions {
```

Comment:
Since this struct isn't user-facing and just an implementation detail, should we move its declaration out of the consumable interface and into the registry itself?